tests/csv/3.0/variantcalling_channels.csv

```diff
@@ -0,0 +1,5 @@
+patient,gender,status,sample,cram,crai
+test,XX,0,sample1,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/cram/test.paired_end.recalibrated.sorted.cram,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/cram/test.paired_end.recalibrated.sorted.cram.crai
+test1,XX,1,sample2,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/cram/test2.paired_end.recalibrated.sorted.cram,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/cram/test2.paired_end.recalibrated.sorted.cram.crai
+test3,XX,0,sample3,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/cram/test.paired_end.recalibrated.sorted.cram,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/cram/test.paired_end.recalibrated.sorted.cram.crai
+test3,XX,1,sample4,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/cram/test2.paired_end.recalibrated.sorted.cram,https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/illumina/cram/test2.paired_end.recalibrated.sorted.cram.crai
```

Comment:
why 2 pairs?

Reply:
I used to make sure that sample3 is not run. I think I am missing a couple of check sthough

Reply:
So basically I justed wanted to make sure we can test all options of the channel magic, that is why two "pairs". Single-normal, single-tumor, and a paired one. Or do you think that is too extensive?

Reply:
Ooooh I see.
May I suggest an alternative solution, not sure if it'll work or not, but could be worth a try
Have 3 different samplesheets:
- single normal
- single tumor
- pair
That way we can use those in different tests.
but call it here in this case with `--input samplesheet1.csv,samplesheet2.csv,samplesheet3.csv`
I know it worked on local with previous sarek version, might be working here as well in this case

Reply:
otherwise, happy with this idea, that's good and smart

Reply:
ah nice, I forgot again about "merging" of input csvs. I will try it then we can just use the ones we have

Reply:
Don't think that works anymore: ```No such file: /Users/monarchy/Projects/Coding/sarek/tests/csv/3.0/recalibrated_germline.csv,tests/csv/3.0/recalibrated_tumoronly.csv,tests/csv/3.0/recalibrated_somatic.csv``` 

with providing it with:
```nextflow run main.nf -profile test,variantcalling_channels,docker --input tests/csv/3.0/recalibrated_germline.csv,tests/csv/3.0/recalibrated_tumoronly.csv,tests/csv/3.0/recalibrated_somatic.csv```

Is that a feature you need again?

Reply:
quotes?
`nextflow run main.nf -profile test,variantcalling_channels,docker --input "tests/csv/3.0/recalibrated_germline.csv,tests/csv/3.0/recalibrated_tumoronly.csv,tests/csv/3.0/recalibrated_somatic.csv"`

Reply:
nope neither single nor double quotes. Trying to find the place in sakre 2.7.1 that implementted this

Reply:
you can stop looking, managed to run:
```
nextflow run . -profile test,docker --input "tests/csv/3.0/fastq_*.csv"
```
There's no `,` involved, just `*` so won't work on https path, just s3 + local

Reply:
hm, ok what works locally now is ```nextflow run main.nf -profile test,variantcalling_channels,docker --input tests/csv/3.0/recalibrated_{germline,tumoronly,somatic}.csv --tools strelka ```

Reply:
it think it is the `if (params.input) csv_file = file(params.input)` line and how this is expanded https://www.nextflow.io/docs/latest/script.html#files-and-i-o

Reply:
yes, we can use globs, so `*` or `{}`, I'd say probably `?`, but not https definitively

Reply:
ah never mind actually doesn't run the processes except for the first file.